addons/point_of_sale/static/src/js/models.js

```diff
@@ -2051,26 +2072,23 @@ exports.Paymentline = Backbone.Model.extend({
             this.trigger('change',this);
         }
     },
-    // returns the payment type: 'cash' | 'bank'
-    get_type: function(){
-        return this.cashregister.journal.type;
-    },
     // returns the associated cashregister
     //exports as JSON for server communication
     export_as_JSON: function(){
+        var amount = this.get_amount();
+        if(this.payment_method.is_cash_count && amount > 0) {
+            amount = amount - this.order.get_change();
+        }
         return {
             name: time.datetime_to_str(new Date()),
-            statement_id: this.cashregister.id,
-            account_id: this.cashregister.account_id[0],
-            journal_id: this.cashregister.journal_id[0],
-            amount: this.get_amount()
+            payment_method_id: this.payment_method.id,
+            amount: amount
         };
     },
     //exports as JSON for receipt printing
     export_for_printing: function(){
         return {
             amount: this.get_amount(),
```

Comment:
we probably want to send the payment_method to the receipt to show that it has been paid by cash or bank

Reply:
the payment method is not being used in the rendering of the receipt. But I can pass the name of the payment method as originally intended in the old code.